libavformat/mux.c

```diff
@@ -163,6 +163,9 @@ int avformat_alloc_output_context2(AVFormatContext **avctx, AVOutputFormat *ofor
                 ret = AVERROR(EINVAL);
                 goto error;
             }
+            /* If format is "mp4" and output format was properly set up, then set call_io_open flag */
```

Comment:
And this would not be needed if we used s->pb condition above.

Am I missing something?

Reply:
@elv-serban my assumption is it should be OK to seek in this case, since we write to a file. Am I correct?